### PR TITLE
added list of attractions by category to Accommodations

### PR DESCRIPTION
**Pull request by shreya317**

### CODE REVIEW

File: app/assets/javascripts/components/accommodations_index.es6 [hunk 1]
```diff
@@ -43,19 +43,75 @@ const AccommodationsIndex = React.createClass({
...
+                  <li><a href="http://www.artic.edu/visit?campaign=chicago%20art%20institute&gclid=CjwKEAiAuea1BRCbn-2n7PbLgEMSJAABQvTTLtn0zA769NOcM7oiFvYx-RLMc_a0owPJK84MM07MLRoCLu3w_wcB" target="_blank">The Art Insitute of Chicago</a></li>
+                  <li><a href="http://www.adlerplanetarium.org/?gclid=CjwKEAiAuea1BRCbn-2n7PbLgEMSJAABQvTT17CaweYWlP3sk8fAnQSrm8dM5SvTLO12CdMff88JRRoC9FTw_wcB" target="_blank">Adler Planetarium</a></li>
+                  <li><a href="http://www.msichicago.org/" target="_blank">Museum of Science & Industry</a></li>
+                  <li><a href="https://www.fieldmuseum.org/" target="_blank">The Field Museum</a></li>
+                </ul>
+
+                <ul>
+                  <h5>Tours / Sightseeing</h5>
+                  <li><a href="http://serverbackup.shorelinesightseeing.com/architecture-tours/" target="_blank">Chicago Architecture Tours</a></li>
+                  <li><a href="http://www.cityofchicago.org/city/en/depts/dca/supp_info/millennium_park.html" target="_blank">Millenium Park</a></li>
```
Comment (pdadhaniya) [inline app/assets/javascripts/components/accommodations_index.es6]: `Millennium`*

File: app/assets/javascripts/components/accommodations_index.es6 [hunk 2]
```diff
@@ -43,19 +43,75 @@ const AccommodationsIndex = React.createClass({
...
+                  <li><a href="http://www.cityofchicago.org/city/en/depts/dca/supp_info/millennium_park.html" target="_blank">Maggie Daily Park</a></li>
```
Comment (pdadhaniya) [inline app/assets/javascripts/components/accommodations_index.es6]: `Daley`*

File: app/assets/javascripts/components/accommodations_index.es6 [hunk 3]
```diff
@@ -43,19 +43,75 @@ const AccommodationsIndex = React.createClass({
...
+                </ul>
+              </div>
+
+              <div className="medium-4 columns">
+                <ul>
+                  <h5>Food / Restaurants</h5>
+                  <li><a href="http://www.loumalnatis.com/locations" target="_blank">Lou Malnatis Pizza</a></li>
```
Comment (pdadhaniya) [inline app/assets/javascripts/components/accommodations_index.es6]: `Malnati's`*

File: app/assets/javascripts/components/accommodations_index.es6 [hunk 4]
```diff
@@ -43,19 +43,75 @@ const AccommodationsIndex = React.createClass({
...
+                  <li><a href="http://pequodspizza.com/chicago/" target="_blank">Pequod's Pizza</a></li>
+                  <li><a href="http://rpmrestaurants.com/rpmitalian/chicago/" target="_blank">RPM Italian</a></li>
+                  <li><a href="http://www.quartinochicago.com/" target="_blank">Quartino</a></li>
+                  <li><a href="http://cumin-chicago.com/index.html" target="_blank">Cumin</a></li>
+                  <li><a href="http://himalayanrestaurant.com/" target="_blank">Himalyan</a></li>
```
Comment (pdadhaniya) [inline app/assets/javascripts/components/accommodations_index.es6]: `Himalayan`*

Also should it be Himalayan Restaurant?

File: app/assets/javascripts/components/accommodations_index.es6 [hunk 5]
```diff
@@ -43,19 +43,75 @@ const AccommodationsIndex = React.createClass({
...
+                <ul>
+                  <h5>Sporting Events</h5>
+                  <li><a href="http://chicago.cubs.mlb.com/schedule/?c_id=chc#y=2016&m=8&calendar=DEFAULT" target="_blank">Chicago Cubs Game</a></li>
+                  <li><a href="http://chicago.whitesox.mlb.com/schedule/index.jsp?c_id=cws#y=2016&m=8&calendar=DEFAULT" target="_blank">Chicago White Sox Game</a></li>
+                </ul>
+
+                <ul>
+                  <h5>Zoos</h5>
+                  <li><a href="http://www.lpzoo.org/" target="_blank">Lincoln Park Zoo</a></li>
+                  <li><a href="https://www.czs.org/Brookfield-ZOO/Home" target="_blank">Brookfrield Zoo</a></li>
```
Comment (pdadhaniya) [inline app/assets/javascripts/components/accommodations_index.es6]: `Brookfield`*

**Comment by shreya317** [comment]
fixed spelling errors @pdadhaniya

**Comment by pdadhaniya** [comment]
:+1:

**Pull request merged**